requirements.txt

```diff
@@ -138,12 +138,12 @@ invenio-oauthclient==1.1.2
 invenio-openaire==1.0.0a15
 invenio-opendefinition==1.0.0a9
 invenio-pidrelations==1.0.0a4
-invenio-pidstore==1.0.0
+invenio-pidstore==1.1.0
 invenio-previewer==1.0.0a11
 invenio-queues==1.0.0a2
 invenio-records==1.3.2
 invenio-records-files==1.0.0a11
-invenio-records-rest==1.6.3
+invenio-records-rest==1.6.6
```

Comment:
Have a look at https://github.com/inveniosoftware/invenio-records-rest/commit/526dcaef46dfe79acf737feee4570fd6e0b25679, I'm not sure what's the effect of it on our error responses (I think we actually have a similar custom logic)

Reply:
This also implies `invenio-base>=1.2.2`. I see [this addition](https://github.com/inveniosoftware/invenio-base/commit/1b16ba66767eb5fa38b5aa8eeeca6f16669253aa), so I would create an issue for replacing `obj_or_import_string` usage through the repo with the one coming from `invenio_base.utils`

Reply:
Don't forget create an issue from this guy. Apparently it's easy to do this from the comment's menu:

![image](https://user-images.githubusercontent.com/1814661/110788076-72dc8480-826e-11eb-8568-dec90c4057b6.png)